mavsdk_tests/.gitignore

```diff
@@ -0,0 +1 @@
+build
```

Comment:
Can this all live one level down within the existing `test/` folder?

Reply:
I guess so, and then it needs to integrate into main build system and all?